app/styles/modules/_custom-rows.scss

```diff
@@ -86,6 +86,16 @@ ul.links {
 }
 
 .choose-what-to-sync-row {
+  &:active{
+    .fxa-checkbox__box-outline{
+      border: 2px solid $checkbox-border-color;
+
+      .fxa-checkbox__tick-outline{
+        background: url("/images/tick.svg");
```

Comment:
> Sounds good, created Issue to track this.

:+1: we can close this pr and move it into there. then the content server PR will be just updating bower.json